tendermint/src/lite_impl/signed_header.rs

```diff
@@ -59,8 +60,18 @@ impl lite::Commit for block::signed_header::SignedHeader {
             );
         }
 
+        // TODO: this last check is only necessary if we do full verification (2/3)
```

Comment:
why it's a TODO?

Reply:
No idea, it was in the code from before.